iteration_rate: 5

Aligngate:
    defaults:
        target_yaw: 3 #Not Sure
        yaw_threshold: 1 #Not Sure

ApproachGate:
  defaults:
    target_surge_time: 50
    speed: 20


ApproachMarker:
  defualts:
    movement_type: 0
    target_surge_time: 100
    speed: 20

StrafeAroundMarker:
  defaults:
    target_sway_time: 30
    sway_speed: 20


ReturnGate:
  defaults:
    target_surge_time: 100
    speed: 20

TurnAroundMarker:
  defaults:
    target_yaw: 90
    yaw_threshold: 10

MovePastMarker:
  defaults:
    target_surge_time: 30
    speed: 20


TurnToGate:
  defaults:
    target_yaw: 180
    yaw_threshold: 10

ReturnToGate:
  defaults:
    target_surge_time: 100
    speed: 20